rpcs3/Emu/RSX/rsx_utils.h

```diff
@@ -182,6 +182,15 @@ namespace rsx
 		return result;
 	}
 
+	// Copy memory in inverse direction from source
```

Comment:
Move this func above the Z index method to keep the morton order functions together (calculate_z_index, convert_linear_swizzle, convert_linear_swizzle_3d)